book/netlify.sh

```diff
@@ -14,33 +14,6 @@ curl -L https://github.com/Michael-F-Bryan/mdbook-linkcheck/releases/download/v$
 unzip mdbook-linkcheck.v$MDBOOK_LINKCHECK_VERSION.x86_64-unknown-linux-gnu.zip -d ~/.cargo/bin
 chmod +x ~/.cargo/bin/mdbook-linkcheck
 
-# ======================================================================
-# The following script automates the deployment of both the latest and a
-# specified older version of the 'salsa' documentation using mdbook
-
-# Store the current branch or commit
-original_branch=$(git rev-parse --abbrev-ref HEAD)
-if [ "$original_branch" == "HEAD" ]; then
-  original_branch=$(git rev-parse HEAD)
-fi
-
-mkdir -p versions  # Create a root directory for all versions
-
-# Declare an associative array to map commits to custom version directory names
-declare -A commit_to_version=( ["$original_branch"]="salsa2022" ["754eea8b5f8a31b1100ba313d59e41260b494225"]="salsa" )
-
-# Loop over the keys (commit hashes or branch names) in the associative array
-for commit in "${!commit_to_version[@]}"; do
-  git checkout $commit
-  mdbook build
-  version_dir="versions/${commit_to_version[$commit]}"
-  mkdir -p $version_dir
-  mv book/html/* $version_dir
-  rm -rf book
-done
-
-# Return to the original branch or commit
-git checkout $original_branch
-
-# Copy _redirects to the root directory
-cp _redirects versions
+mdbook build
+mkdir versions
+mv book/html/* versions
```

Comment:
Only caveat here: I feel like maybe we should keep the older book for now. But I am happy to not have the complexity, maybe we can just "clone" the old book into the new book as a subdirectory for the time being or something.

Reply:
(Doesn't have to be done in this PR)

Reply:
Filed https://github.com/salsa-rs/salsa/issues/503